packages/client/src/components/MyContracts.tsx

```
@@ -0,0 +1,238 @@
+
+        const profileData = JSON.parse(contractorProfile);
+        const email = profileData.email;
+
+        if (!email) {
+          setError("No contractor email found. Please complete your profile first.");
+          return;
+        }
+        
+        const response = await getContractorByEmail({ email });
+        
+        if (response && response.contractor && response.contractor.contractorId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)